src/main/java/ratpack/pac4j/RatpackPac4j.java

```diff
@@ -213,7 +208,8 @@ public static Handler authenticator(String path, ClientsProvider clientsProvider
           try {
             boolean authorized = true;
             for (Authorizer a : authorizerList) {
-              if (a != null && !a.isAuthorized(webContext, Arrays.asList(userProfile))) {
+              if (a != null && !a.isAuthorized(webContext, webContext.getSessionStore(),
+                  Collections.singletonList(userProfile))) {
```

Comment:
Not a big difference, but you could consider using `ImmutableList.of` instead of `Collections.singletonList`.

Reply:
That would only work if there's a dependency on Guava, correct?

Reply:
Yes. However, this file was already using ImmutableList and Ratpack always depends on Guava.